hknweb/events/templates/events/show_details.html

```diff
@@ -64,8 +64,10 @@ <h1 id="event-detail-title"> {{ event.name }} </h1>
         <input type="submit" value="RSVP" />
       </form>
     {% endif %}
-
-    {% if rsvpd %}
+    
+    {% if waitlisted %}
+      <p class="rsvp-list">You are waitlisted. You are at position {{waitlist_position}}</p>
+    {% elif rsvpd %}
```

Comment:
Also, should we display the waitlist and waitlist size? It would be useful to the hosts of the event, for one.